githubapp/scheduler.go

```diff
@@ -119,11 +122,16 @@ func WithSchedulingMetrics(r metrics.Registry) SchedulerOption {
 		metrics.NewRegisteredFunctionalGauge(MetricsKeyActiveWorkers, r, func() int64 {
 			return atomic.LoadInt64(&s.activeWorkers)
 		})
+
+		// use sample values from metrics.NewTimer which match those used by UNIX load averages
+		s.eventAge = metrics.NewRegisteredHistogram(MetricsKeyEventAge, r, metrics.NewExpDecaySample(1028, 0.015))
```

Comment:
nit: pull out these values into constants?

Reply:
curious: why not have this be a gauge, always showing the current age, and then letting the downstream data stores let you get higher-order values out of it.

Reply:
I went with a histogram due to reporting intervals. For most of our apps, we loop through metrics every 10s and send the current values to Datadog. With a gauge, that means you see the age of the most recently handled event in the last 10s and nothing else. In the case of a big queue, maybe that's often enough, but I thought having the extra granularity would be better.

I'll also move the sample parameters to constants.